react/components/LayoutContainer.tsx

```diff
@@ -43,8 +44,11 @@ const Container: FunctionComponent<ContainerProps> = ({
     if (elements === '__children__') {
       return <>{children}</>
     }
+    const id = Buffer.from(elements, 'binary').toString('base64')
+    const classNameWithId = `${isRow ? '' : className + ' '}id-${id}`
```

Comment:
Hi @arturmagalhaesjr , I would still go for the css handles solution, there's no problem to add the lib for the first time there and increase the PR size, the most important is to keep our standarts/patterns in new classes, so, I would go for this implementation:

https://developers.vtex.com/docs/apps/vtex.css-handles

The first example of usage with useCssHandles is already enough for that, something like:

const CSS_HANDLES = ['container'] as const
const { handles } = useCssHandles(CSS_HANDLES, { classes })

...
<div className={handles.container}>

Reply:
Yes, there's no problem with adding this new dependency, the main point here is to keep our code standards if necessary, and in this case, we should follow our patterns since it's necessary.

It's better to do it now than in the future if we want to add a new CSS class anywhere else since this part of the code will be forgotten

Reply:
@arturmagalhaesjr , did you have time to review it?